Formula/c/cekit.rb

```diff
@@ -17,7 +17,7 @@ class Cekit < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "1953c1205a6b3ff72285b3d7899973a7d8c96db7d33cc6530c2928c9b73ae882"
   end
 
-  depends_on "python-setuptools" => :build
+  depends_on "libyaml"
```

Comment:
Just raised over [here](https://github.com/Homebrew/brew/pull/16718#issuecomment-1955587468), wondering if this is something we should include in the pyyaml audit? (e.g. include this if either `ruamel.yaml` or `PyYAML` are a Python dep)

Reply:
We can. I commented in the other PR as well. We should get this merged though, since we won't be able to merge in the new Rubocop while this remains an issue.